.github/workflows/release.yml

```diff
@@ -49,15 +49,29 @@ jobs:
       - name: Build and push
         run: bash .ci-scripts/release/build-docker-images-on-release.bash
 
+  update-latest-release-tag:
+    name: Update latest-release tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Tag
+        run: |
+          git tag --force latest-release
+          git push --force "https://${TOKEN}@github.com/${GITHUB_REPOSITORY}.git" latest-release
```

Comment:
where does `GITHUB_REPOSITORY` come from?

Reply:
Its defined by GH for every CI run.

See https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables

That series of Gitness right there was in the existing release script that is going away so we can use the more general bot.